src/downscaling_stats/downscaling.f90

```diff
@@ -902,6 +905,7 @@ subroutine downscale_analog_regression(x, atm, obs, output_coeff, output, error,
                 ! Check for severe errors in the logistic regression.  This can happen with some input data.
                 ! If it fails, fall back to the analog exceedence calculation
                 if (ieee_is_nan(logistic)) then
+                ! if (logistic /= logistic) then
```

Comment:
why this? Maybe just remove the commented out line

Reply:
The commented line was because I was having difficulty using the ieee math with one compiler or another (I think it was gfortran 8 something) and wanted a quick way to switch between the cheap hack and the correct way to do this.  I think you're right though, this doesn't need to be in the committed code